templates/ts-apollo-mongodb-backend/.graphqlrc.yml

```diff
@@ -2,6 +2,21 @@
 ## Please review configuration and adjust it for your own project
 schema: ./src/schema/schema.graphql
 extensions:
+  graphback:
+    model: ./model/**/*.graphql
```

Comment:
We should load the model from this in the template, otherwise config needs to be maintained in multiple places

Reply:
This is only needed for the `graphback-cli`

Reply:
Yep, but this still means that path is declared in multiple places - in code and in this

Reply:
Sorry I completely misunderstood your initial comment. You are right. That could improve things. 

Let me push something for you to review.